lib/pool_sup.ex

```diff
@@ -40,6 +40,18 @@ defmodule PoolSup do
   #
   # client API
   #
+  @doc """
+  Returns a specification to start this module under a supervisor.
```

Comment:
Let's change this doc comment (and the corresponding one for `Multi`) to something like:
```
Returns a child specification to be used when it's not fully specified by the parent supervisor.
```

Reply:
fixed in d429f41